api/server/models/user.js

```diff
@@ -0,0 +1,121 @@
+var GENERIC = require('../modules/generic');
+var lodash = require('lodash');
+
+module.exports = function (User) {
+    User.register_login = function (action, action_type, social_id, platform, device_id, token, email_id, name, password, currentTimestamp, currentDate, currentDateTimeDay, callback) {
+        name = name.toLowerCase();
```

Comment:
can you put validations here. 
e.g is email, name etc are null or empty it will return error message from here itself